docs/ruleset_functions/get_hvac_sys_and_assoc_zones_largest_exhaust_source.md

```diff
@@ -7,7 +7,7 @@
 - **hvac_sys**: The hvac system object for which the largest exhaust source is being determined.
 
 **Returns:**  
-- **get_hvac_sys_and_assoc_zones_largest_exhaust_source**: Returns a list with the sum of the hvac fan system exhaust fan cfm values, the maximum zone level exhaust source across the zones associated with the HVAC system, the number of exhaust fans associated with the hvac fan system, and the maximum cfm of all of the exhaust fans associated with the hvac system fan system [hvac_sys_exhaust_cfm_sum, maximum_zone_exhaust, num_hvac_exhaust_fans, maximum_hvac_exhaust]. This is for evaluating G3.1.2.10 exception 6 which states "Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. "  
+- **get_hvac_sys_and_assoc_zones_largest_exhaust_source**: Returns a dict with the sum of the hvac fan system exhaust fan cfm values, the maximum zone level exhaust source across the zones associated with the HVAC system, the number of exhaust fans associated with the hvac fan system, and the maximum cfm of all of the exhaust fans associated with the hvac system fan system {"hvac_sys_exhaust_cfm_sum": 3000, "maximum_zone_exhaust": 500, "num_hvac_exhaust_fans": 2, "maximum_hvac_exhaust": 1000}. This is for evaluating G3.1.2.10 exception 6 which states "Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. "  
```

Comment:
The units should not appear in the keys of the returned dictionary.  This must serve both IP and SI unit systems. So, `hvac_sys_exhaust_cfm_sum` should be `hvac_sys_exhaust_flow_sum` (or if you want to be more verbose, you could use `flow_rate` instead of `flow`.  Also, "maximum cfm" should be replaced with "maximum flow rate" in the accompanying text.

Should `maximum_zone_exhaust` be `maximum_zone_exhaust_flow`? and similarly for `maximum_hvac_exhuast`?

I don't think it is a good idea to include example values in the dictionary here. First, there is no mention that this is just an example.  Taken literally, this indicates that this particular dictionary is always returned. Second, the values are not numbers, they are Pint Quantities. I suggest wording something like:
```
Returns a dict with the following form:
{
   "hvac_sys_exhaust_flow_sum": the sum of the hvac fan system exhaust fan flow rates,
   "maximum_zone_exhaust_flow": the maximum zone level exhaust source across the zones associated with the HVAC system,
   "num_hvac_exhaust_fans": the number of exhaust fans associated with the hvac fan system,
   "maximum_hvac_exhaust_flow": the maximum flow rate of all of the exhaust fans associated with the hvac system fan system
}
```